controllers/numaresourcesoperator_controller.go

```diff
@@ -198,8 +202,10 @@ func (r *NUMAResourcesOperatorReconciler) reconcileResource(ctx context.Context,
 
 	daemonSetsInfo, err := r.syncNUMAResourcesOperatorResources(ctx, instance, mcps)
 	if err != nil {
+		r.Recorder.Eventf(instance, corev1.EventTypeWarning, "FailedRTECreate", "Failed to create Resource-Topology-Exporter DaemonSet: %v", err)
 		return ctrl.Result{}, status.ConditionDegraded, errors.Wrapf(err, "FailedRTESync")
 	}
+	r.Recorder.Eventf(instance, corev1.EventTypeNormal, "SuccessfulRTECreate", "Created Resource-Topology-Exporter DaemonSet: %s", daemonSetsInfo[0].String())
```

Comment:
if we have more than an option (we do: daemonSetsInfo is a slice) and no option is clearly more important than others (it is not) better to log none of them. Hence
"Created Resource-Topology-Exporter DaemonSets"

Reply:
This raises another question (which is unnecessarily related to this PR directly) why we should have a slice of DS when we only have one?
IIRC how DS is working, we have a single DS which makes sure to deploy an RTE pod on every single worker node.

Reply:
we have a 1:1 between node groups and daemonsets. In the test suite/CI we create only one node group, because it was the simplest test. This is actually a test gap.

Reply:
Right! I forgot about the node groups. Thanks!

Reply:
> This is actually a test gap
IMHO it is extensive enough to be tested by QE

Reply:
well, it surely doesn't belong in this PR :)